.github/workflows/binaries.yml

```
@@ -0,0 +1,55 @@
+name: Build binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+
+jobs:
+  binaries:
+    name: Build lodestar binaries
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            build: |
+              npx caxa -m "This may take a while to run the first time, please wait..." -e "dashboards/**" -e "docs/**" -D -p "yarn install --production" --input . --output "lodestar" -- "{{caxa}}/node_modules/.bin/node" "--max-old-space-size=8192" "{{caxa}}/node_modules/.bin/lodestar"
```

what about more specific message? wondering if we should communicate that our binary is just fake :joy: 

```suggestion
              npx caxa -m "Unpacking Lodestar binary, please wait..." -e "dashboards/**" -e "docs/**" -D -p "yarn install --production" --input . --output "lodestar" -- "{{caxa}}/node_modules/.bin/node" "--max-old-space-size=8192" "{{caxa}}/node_modules/.bin/lodestar"
```